colossalai/fx/codegen/activation_checkpoint_codegen.py

```diff
@@ -1,14 +1,19 @@
+import colossalai
 import torch
 from typing import List, Callable, Any, Tuple, Dict
 
 try:
     from torch.fx.node import Node, Argument, map_arg, _type_repr, _get_qualified_name
-    from torch.fx.graph import _Namespace, PythonCode, _custom_builtins, _is_from_torch, _format_target, magic_methods, CodeGen, _origin_type_map, inplace_methods
+    from torch.fx.graph import _Namespace, PythonCode, _custom_builtins, _is_from_torch, _format_target, magic_methods, CodeGen, _origin_type_map, inplace_methods, _CustomBuiltin
     CODEGEN_AVAILABLE = True
+    # set _custom_builtins here so that we needn't import colossalai in forward
+    _custom_builtins["colossalai"] = _CustomBuiltin("import colossalai", colossalai)
```

Comment:
We should add custom builtins when we actually want to use the codegen. Otherwise, this will take effect upon module import even though we may not need activation checkpoint in our code at all.

Reply:
okay, I will fix it